src-vue/src/plugins/store.ts

```diff
@@ -127,7 +127,26 @@ export const store = createStore<FlightCoreStore>({
                 }
             }
         },
-        async launchGame(state: any) {
+        async launchGame(state: any, no_checks = false) {
+            let game_install = {
+                game_path: state.game_path,
+                install_type: state.install_type
+            } as GameInstall;
+
+            if (no_checks) {
+                await invoke("launch_northstar_caller", { gameInstall: game_install })
+                    .then((message) => {
+                        console.log("Launched with bypassed checks");
```

Comment:
Before you complain about leftover console logs again @Alystrasz, I'm thinking maybe we should have one here so that once we actually create log files or something for errors we see if the user actually tried to launch with bypassing checks ^^